src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Commands/AddCohort/AddEmptyCohortHandler.cs

```diff
@@ -0,0 +1,59 @@
+﻿using MediatR;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.CommitmentsV2.Data;
+using SFA.DAS.CommitmentsV2.Domain.Interfaces;
+using SFA.DAS.Encoding;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.CommitmentsV2.Application.Commands.AddCohort
+{
+    public class AddEmptyCohortHandler : IRequestHandler<AddEmptyCohortCommand, AddCohortResult>
+    {
+        private readonly Lazy<ProviderCommitmentsDbContext> _dbContext;
+        private readonly ILogger<AddCohortWithOtherPartyHandler> _logger;
+        private readonly IEncodingService _encodingService;
+        private readonly ICohortDomainService _cohortDomainService;
+
+        public AddEmptyCohortHandler(
+            Lazy<ProviderCommitmentsDbContext> dbContext,
+            IEncodingService encodingService,
+            ILogger<AddCohortWithOtherPartyHandler> logger,
```

Comment:
should be ILogger<AddEmptyCohortHandler>

Reply:
done.